src/Chapter2.hs

```diff
@@ -741,7 +777,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains number lists = filter (\list -> elem number list) lists
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function as well  🙂 
Here you have two opportunities for eta-reduction: the last argument to `filter` and the lambda
But your implementation is already great 💯